README.md

```diff
@@ -81,30 +93,46 @@ Compiles to HTML:
 <!DOCTYPE html>
 <html>
   <head>
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>This is a title</title>
+    <style>
+      h1 {color: red;}
+    </style>
   </head>
   <body>
+    <!--Multi-line comment
+      <span>this is wrapped in a comment</span>
+    -->
+    <!--[if IE]>
+      <p>Dat browser is old.</p>
+    <![endif]-->
+    <h1>This is a slang file</h1>
     <span id="some-id" class="classname">
       <div id="hello" class="world world2">
         <span>
           <span data-some-var="hello world haha" two-attr="fun">and a hello</span>
           <span>
             <span class="deep_nested">
-              #{Process.pid}
+              3979
               text node
-              other text node
+              other text node 
             </span>
           </span>
         </span>
-        <span class="alongside" pid="#{Process.pid}">
-          <custom-tag id="with-id" pid="#{Process.pid}">
```

Comment:
So I added some more things to the slang side and then copied over the result... Interpolation still works. I just didn't notice that it stayed in the html side since it said "Compiles to HTML". I assumed it was the output not ecr. Sorry. I'll fix that.